src/parse_file.py

```diff
@@ -104,8 +104,8 @@ def import_from_columns(self, file):
     separator = columns_params.get_string("separator").replace(" ", "")
     skip_rows = columns_params.get_int("skip-rows")
     with file_io.FileLikeWrapper.new_for_file_readonly(file) as wrapper:
-        for index, line in enumerate(wrapper.wrap_text()):
-            if index <= skip_rows:
+        for index, line in enumerate(wrapper.wrap_text(), -skip_rows):
+            if index < 0:
```

Comment:
Had to look twice to see why this unusual logic is used, but it makes line 118 cleaner. So seems more robust.